{"name": "test_unsuccessful_login[chrome-ft-34#435;'/.]", "status": "passed", "attachments": [{"name": "log", "source": "dce36e79-96bf-4e65-8a80-bcb86b895559-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "'ft'"}, {"name": "password", "value": "'34#435;'/.'"}], "start": 1622284081917, "stop": 1622284082466, "uuid": "2492c401-3aff-470e-b083-4dd4b0cca5c4", "historyId": "4b7838c67916d5b4e988ff2db08a9fa0", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "story", "value": "Cloud Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "22520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}